images/nginx/Dockerfile

```
@@ -1,7 +1,7 @@
 ARG IMAGE_REPO
 FROM ${IMAGE_REPO:-lagoon}/commons as commons
-# alpine 3.11 per https://github.com/openresty/docker-openresty/blob/master/alpine/Dockerfile
-FROM openresty/openresty:1.15.8.2-alpine
+# Alpine 3.11 per https://github.com/openresty/docker-openresty/blob/master/alpine/Dockerfile#L5
+FROM openresty/openresty:1.17.8.2-1-alpine
```

```suggestion
FROM openresty/openresty:1.17.8.2-alpine
```
Suggest to pin to the minor release tag to ensure we get any patches that come through

---

Good call, I force pushed a new commit.

---

To a tag that doesn't exist yet :facepalm:

---

doh. the `.3` tag exists in OpenResty, just no in DockerHub. Perhaps `openresty/openresty:1.17.8.2-alpine` ?